library/alloc/src/fmt.rs

```diff
@@ -109,6 +109,16 @@
 //! parameters (corresponding to `format_spec` in [the syntax](#syntax)). These
 //! parameters affect the string representation of what's being formatted.
 //!
+//! The colon `:` in format syntax divides indentifier of the input data
+//! and the formatting options, the colon itself does not change anything, only introduces
+//! the options.
```

Comment:
This looks fine to me but could you rewrap this text? Looks like this file uses 80 characters.